lib/Test/Mojo.pm

```diff
@@ -1071,6 +1083,20 @@ Check response status for exact match.
 
 Opposite of L</"status_is">.
 
+=head2 status_like
+
+  $t = $t->status_like(qr/^4/);
+  $t = $t->status_like(qr/^4/, 'some kind of error');
+
+Check response status for regex match.
+
+=head2 status_unlike
+
+  $t = $t->status_unlike(qr/^3/);
+  $t = $t->status_unlike(qr/^3/, 'not any kind of redirect');
+
+Check response status for B<not> matching a regex.
```

Comment:
Seems inconsistent with other method descriptions of the same type.

Reply:
Fixed